src/s390_ecc.c

```diff
@@ -646,7 +742,6 @@ unsigned int ecdsa_sign_sw(const ICA_EC_KEY *privkey,
 {
 	int rc = 1;
     EC_KEY *a = NULL;
-    BIGNUM* xa=NULL; BIGNUM* ya=NULL;
     BIGNUM* r=NULL; BIGNUM* s=NULL;
```

Comment:
Later in this function there is a line
  ECDSA_SIG_get0(sig, (const BIGNUM**)&r, (const BIGNUM **)&s);
can you please change this to
  ECDSA_SIG_get0(sig, &r, &s);